roles/confluent.control_center/tasks/main.yml

```diff
@@ -14,89 +14,43 @@
   when: ansible_os_family == "Debian"
 
 # Configure environment
-- name: control center create group
+- name: Create Control Center Group
   group:
     name: "{{control_center.group}}"
-- name: control center create user
+
+- name: Create Control Center User
   user:
     name: "{{control_center.user}}"
     comment: "Control Center User"
     system: yes
     group: "{{control_center.group}}"
-- name: Change keytabs ownership
+
+- name: Create Keytabs Directory
   file:
     path: "{{kerberos.keytab_dir}}"
     group: "{{control_center.group}}"
     mode: '650'
     recurse: yes
-- name: control center create config
-  template:
-    src: control-center.properties.j2
-    dest: "{{control_center.config_file}}"
-    mode: 0640
-    owner: "{{control_center.user}}"
-    group: "{{control_center.group}}"
-  when: security_mode == "plaintext"
-  notify:
-    - restart control center
-- name: control center create config
-  template:
-    src: control-center_ssl.properties.j2
-    dest: "{{control_center.config_file}}"
-    mode: 0640
-    owner: "{{control_center.user}}"
```

Comment:
The request is to modify the way kerberos security is handled.  Why are we refactoring how all templates work?

Reply:
template refactoring is in the description